config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='utf-8'?>
+<widget id="com.myapp.mobile" version="2.25" xmlns="http://www.w3.org/ns/widgets">
+  <name>Homework Checklist</name>
+  <description>Homework Checklist</description>
```

Comment:
This does not look like the config.xml that phoneGap desktop generates. That is your issue.